### PR TITLE
kommander / opsportal: Bump kommander-ui to 6.85.1

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

bug

**What this PR does/ why we need it**:

Bump kommander-ui to fix non-konvoy cluster platform services page https://github.com/mesosphere/kommander-ui/pull/3292. Would be great to include this in the next kommander 1.3.1 patch release.

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-74138

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
fix(kommander): Fix empty non-Konvoy cluster Platform Services tab 
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Comment by gracedo** [comment]
@TattdCodeMonkey @natmegs We discussed back at the last release that we would release this ui fix in a patch release - I noticed there is a jump in version from 6.84.3 to 6.85.1 which includes this fix. Can you guys double check what other changes would be included in this version jump and whether it's okay to include them in the next kommander/konvoy patch release?

**Review by TattdCodeMonkey — APPROVED** [review]

File: stable/kommander/Chart.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ maintainers:
   - name: gracedo
   - name: dkoshkin
 name: kommander
-version: 0.13.36
+version: 0.13.37
```
Comment (joejulian) [inline stable/kommander/Chart.yaml:12]: This includes a minor version update to kommander-ui so the chart version should also have a minor version bump.
```suggestion
version: 0.14.0
```

**Review by joejulian — CHANGES_REQUESTED** [review]

File: stable/kommander/Chart.yaml (hunk 1 above)
Comment (s12chung) [reply stable/kommander/Chart.yaml:12]: makes sense.

Comment (gracedo) [reply stable/kommander/Chart.yaml:12]: makes sense to me too. thus far kommander has (more or less) bumped patch versions for changes within a kommander release, then bumped the minor when the release goes out. I think it makes sense to actually follow semver, we just need to get everyone on the same page so it's consistent

**Review by joejulian — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Pull request merged**